src/components/HexGrade/HexGrade.tsx

```diff
@@ -65,8 +73,9 @@ const HexGrade: React.FC<HexGradeProps> = ({
       : colors.graphite5H;
 
   return (
-    <svg
+    <StyledSVG
       height={size}
+      margin={margin}
```

Comment:
I'm just worried that here the `auto` value will be missed a lot :/

Reply:
Eventually yeah but FlexContainer is usually good enough for now